docker/debian/Dockerfile

```diff
@@ -209,7 +209,7 @@ ENV LC_ALL "en_US.UTF-8"
 ENV GRASS_SKIP_MAPSET_OWNER_CHECK 1
 
 # Create generic GRASS GIS lib name regardless of version number
-RUN ln -sf /usr/local/grass80 /usr/local/grass
+RUN ln -sf /usr/local/grass81 /usr/local/grass
```

Comment:
Candidate for a bigger change? Something like `ln -sf /usr/local/grass*` would work too, no? There is only one GRASS in that Docker image. (Same for all Dockerfiles.)

Or ever bigger change, why it is installed to grassXY in the first place? I have `/usr/local/share/zsh/` and `/usr/local/lib/R` there (although I have `/usr/local/lib/python2.7/` there too). Obviously, at least this piece is outside of scope of this PR and thus just a side note.

Reply:
> Candidate for a bigger change? Something like `ln -sf /usr/local/grass*` would work too, no? There is only one GRASS in that Docker image. (Same for all Dockerfiles.)

We had that already in the past and it failed (don't recall well why, perhaps it let to a recursive effect somehow). See f1a46d63fb3628d7ea506a146390f21d681c6d0d for the revert in last summer.

> Or ever bigger change, why it is installed to grassXY in the first place? I have `/usr/local/share/zsh/` and `/usr/local/lib/R` there (although I have `/usr/local/lib/python2.7/` there too). Obviously, at least this piece is outside of scope of this PR and thus just a side note.

The point is: how to then have distinct GRASS versions installed? But perhaps for another PR...

Reply:
> > Candidate for a bigger change? Something like `ln -sf /usr/local/grass*` would work too, no? There is only one GRASS in that Docker image. (Same for all Dockerfiles.)
> 
> We had that already in the past and it failed (don't recall well why, perhaps it let to a recursive effect somehow). See [f1a46d6](https://github.com/OSGeo/grass/commit/f1a46d63fb3628d7ea506a146390f21d681c6d0d) for the revert in last summer.
> 
> > Or ever bigger change, why it is installed to grassXY in the first place? I have `/usr/local/share/zsh/` and `/usr/local/lib/R` there (although I have `/usr/local/lib/python2.7/` there too). Obviously, at least this piece is outside of scope of this PR and thus just a side note.
> 
> The point is: how to then have distinct GRASS versions installed? But perhaps for another PR...

Agree this is a question for another PR, but we have already stripped the version from grass executable. For multiple version-installations alternative `--prefix` and `--exec-prefix` may be set.

Reply:
> > ...Something like `ln -sf /usr/local/grass*` would work too, no?...
> 
> ...it failed (don't recall well why, perhaps it let to a recursive effect somehow). See [f1a46d6](https://github.com/OSGeo/grass/commit/f1a46d63fb3628d7ea506a146390f21d681c6d0d)...

Hm, I don't see a PR or an error related to that, so hard to say. It worked in other places at least for the executable. I have seen (less general) `grass??` elsewhere, so perhaps you or someone else successfully addressed the issue, whatever it was, this way.

> > ...why it is installed to grassXY in the first place? I have `/usr/local/share/zsh/` and `/usr/local/lib/R` there...
> 
> The point is: how to then have distinct GRASS versions installed?

Definitively another PR. But what is the use case when you need to have two GRASS versions installed side by side like this? Certainly, it is not these Dockerfiles. I would think that developers don't install the software with sudo to a system location like this either, or? PATH/environment modification and aliases can be used to have custom setups of multiple versions of the same software. Environment modification is exactly what tools like Python virtual environments are using.